module/programs/helper.js

```diff
@@ -511,6 +511,9 @@ module.exports = class ProgramsHelper {
         if( !queryData.success ) {
           return resolve(queryData);
         }
+        if(programId !== ""){
```

Comment:
@ankitpws add a condition if someone passes as an ObjectId we should not convert. Function should be generic to accept string or object

Reply:
convertStringToObjectId will internally check